code/code.py

```diff
@@ -15,6 +15,7 @@
     # 'cplusplus': 'cpp hpp',
     "csharp": "cs",
     "css": "css",
+    "scss": "scss",
```

Comment:
Can you delete `sass` on line 39 below? It also looks like this is a sorted list so we may want to move this down.